packages/components/src/form-token-field/suggestions-list.tsx

```diff
@@ -122,6 +123,24 @@ export function SuggestionsList< T extends string | { value: string } >( {
 					}
 				);
 
+				let output: ReactNode;
+
+				if ( typeof renderSuggestion === 'function' ) {
+					output = renderSuggestion( suggestion );
```

Comment:
If we pass `suggestion` as a property, this would enable us to pass more data in the future without introducing breaking changes to this function's signature (for example, we may want to pass `matchText` or any other piece of data)

```suggestion
					output = renderSuggestion( { suggestion } );
```

WDYT?

Reply:
Yes, I always prefer and like that approach but I saw that `displayTransform` callback has a similar signature, thus I made it the same.

Reply:
I see, that makes sense. I'm still tempted to go for the suggested object-like approach (and potentially look into introducing that same signature to the `displayTransform` at some point) — @mirka , WDYT ?

Reply:
Yes, I have changed it to object-like approach.